module.exports = {
  UNKNOWN_ERROR: 'Unknown error',
  UNAUTHORIZED_ERROR: 'User in not authorized',
  INVALID_CREDENTIALS: 'Invalid credentials',
  USER_ALREADY_EXISTS: 'User with this username already exists',
  USER_NOT_EXIST: 'User does not exist ',
  CONTACT_ALREADY_EXISTS: 'User already added to contacts',
  CONTACT_NOT_EXIST: 'No such user in contacts',
  CONTACT_ADD_YOURSELF: 'Cannot add yourself',
  CONTACT_REMOVE_YOURSELF: 'Cannot remove yourself',
  NO_ACCESS: 'No access',
  CHAT_EXISTS: 'Chat already exists',
  CHAT_YOURSELF: 'Cannot chat yourself',
  DELETE_PRIVATE_CHAT: 'Cannot delete private chat',
  ADD_TO_PRIVATE_CHAT: 'Cannot add users to private chat',
  NO_USER_IN_CHAT: 'No such user in chat',
  REMOVE_FROM_PRIVATE_CHAT: 'Cannot remove users from private chat',
  ALREADY_IN_CHAT: 'User already in chat',
  CHAT_ADD_YOURSELF: 'Cannot add yourself',
  CHAT_REMOVE_YOURSELF: 'Cannot remove yourself',
  LEAVE_FROM_PRIVATE_CHAT: 'Cannot leave from private chat',
  NOT_FOUND: 'User not found',
};
